src/distrs/tdist.jl

```diff
@@ -22,3 +8,67 @@ function tdistlogpdf(ν::T, x::T) where {T<:Real}
     νp12 = (ν + 1) / 2
     return loggamma(νp12) - (logπ + log(ν)) / 2 - loggamma(ν / 2) - νp12 * log1p(x^2 / ν)
 end
+
+function tdistcdf(ν::T, x::T) where T<:Real
+    if isinf(ν)
+        return normcdf(x)
+    elseif x < 0
+        return fdistccdf(one(ν), ν, x*x)/2
+    else
+        return 1 - fdistccdf(one(ν), ν, x*x)/2
+    end
+end
+tdistcdf(ν::Real, x::Real) = tdistcdf(map(float, promote(ν, x))...)
+
+tdistccdf(ν::Real, x::Real) = tdistcdf(ν, -x)
+
+function tdistlogcdf(ν::T, x::T) where T<:Real
+    if isinf(ν)
+        return normlogcdf(x)
+    elseif x < 0
+        ret = fdistlogccdf(one(ν), ν, x*x)
+        return ret - log(2*one(ret))
```

Comment:
FWIW I believe this should also work as expected and avoids needing to actually compute the log:
```suggestion
        return ret - logtwo
```